src/main/java/hu/blackbelt/judo/dao/api/DAO.java

```diff
@@ -211,6 +211,20 @@ public interface DAO<ID> {
      */
     Payload create(EClass clazz, Payload payload, QueryCustomizer<ID> queryCustomizer);
 
+
+    /**
+     * Create a new instances of a given mapped transfer object type.
```

Comment:
instance or instances?

Reply:
It returns collection, means intances :)

Reply:
[5a48de3](https://github.com/BlackBeltTechnology/judo-dao-api/pull/63/commits/5a48de3d61be56c1e58e3098a664830227edda0d)